concourse/pipelines/create-bosh-concourse.yml

```diff
@@ -86,6 +86,13 @@ resources:
       branch: ((branch_name))
       commit_verification_keys: ((gpg_public_keys))
 
+  - name: paas-trusted-people
+    type: git
+    source:
+      uri: https://github.com/alphagov/paas-trusted-people.git
+      branch: master
+      commit_verification_keys: ((gpg_public_keys))
```

Comment:
🤔  this should have a tag filter, otherwise changes will go straight to prod.

Reply:
Although paas-bootstrap doesn't auto-trigger, so less of an issue, but still.

Reply:
I think we should use a tag here

Reply:
This is a moot point now that it's a private repo - the PR's been updated so the YAML file in paas-trusted people is uploaded to S3 instead of pulled from the repo by the pipeline.